spec/game_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'game'
```

Comment:
Great use of describe and context within these tests, it made it clear to understand the functionality of each test.